.github/workflows/installers-conda.yml

```diff
@@ -48,13 +48,13 @@ concurrency:
 name: Create conda-based installers for Windows, macOS, and Linux
 
 env:
-  USE_SUBREPOS: ${{ github.event_name == 'pull_request' || github.event == 'schedule' || (github.event_name == 'workflow_dispatch' && ! inputs.pre) }}
   IS_RELEASE: ${{ github.event_name == 'release' }}
-  IS_PRE: ${{ github.event_name == 'workflow_dispatch' && inputs.pre }}
+  ENABLE_SSH: ${{ github.event_name == 'workflow_dispatch' && inputs.ssh }}
   BUILD_MAC: ${{ github.event_name != 'workflow_dispatch' || inputs.mac }}
   BUILD_LNX: ${{ github.event_name != 'workflow_dispatch' || inputs.linux }}
   BUILD_WIN: ${{ github.event_name != 'workflow_dispatch' || inputs.win }}
-  ENABLE_SSH: ${{ github.event_name == 'workflow_dispatch' && inputs.ssh }}
+  USE_SUBREPOS: ${{ github.event_name == 'schedule' || github.event_name == 'pull_request' || (github.event_name == 'workflow_dispatch' && ! inputs.pre) }}
+  NOTARIZE: ${{ github.event_name == 'schedule' || github.event_name == 'release' || (github.event_name == 'workflow_dispatch' && inputs.pre) }}
```

Comment:
I just noticed that this depends on `inputs.pre`, but you removed it above. Should it be fixed?

Reply:
I removed the convenient environment variable `IS_PRE` because it is no longer needed. However, the workflow-dispatch input `inputs.pre` is still needed.